package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "build:jlab": "cd .. && jlpm run build",
+    "build:jlab": "cd $JLAB_HOME && jlpm run build",
     "build:benchmark": "tsc",
-    "start:jlab": "fkill -s :9999 && cd .. && jupyter lab --dev --no-browser --port 9999 --ServerApp.password= --ServerApp.token=",
+    "start:jlab": "fkill -s :9999 && jupyter lab --dev --no-browser --notebook-dir=${PWD} --port 9999 --ServerApp.password= --ServerApp.token=",
```

Comment:
Ah so you don't need to be in the jupyter lab source directory to run `jupyter lab --dev`?

Reply:
When I try starting jupyterlab locally with `--dev-mode` it doesn't seem to work unless I am in the jupyterlab source directory?

<img width="977" alt="Screen Shot 2020-08-10 at 11 24 31 AM" src="https://user-images.githubusercontent.com/1186124/89799913-1197db80-dafc-11ea-8e56-e19c6dd1ba53.png">

Reply:
no, because of https://github.com/jupyterlab/jupyterlab/blob/2b225c5c85e115621bb7d3e6424db76078025d9b/jupyterlab/commands.py#L42 and also other mecanisms (I did the exercise some time ago, but this is a whole logic to follow)

Reply:
he he, that is side effect of the recent merge for jupyter_server. You have multiple tabs with jlab. If you close them, it should work.

Reply:
Oh good point! Thanks for the tip :)